composer.json

```diff
@@ -55,7 +55,8 @@
         "psr/cache-implementation": "To use metadata caching.",
         "symfony/cache": "To have metadata caching when using Symfony integration.",
         "symfony/config": "To load XML configuration files.",
-        "symfony/twig-bundle": "To use the Swagger UI integration."
+        "symfony/twig-bundle": "To use the Swagger UI integration.",
+        "swagger-api/swagger-ui": "^2.2"
```

Comment:
I added a warning in case swagger doesn't exist, maybe you want to check the message I put there? 

https://github.com/api-platform/core/pull/768/files#diff-46d58047a477563966bc28138a2eb7ceR30

> but displaying an error pointing to the doc explaining that the Swagger UI package and the related Composer script must be installed in order to have the UI?

Where should this go? I'm a bit lost with swagger as I didn't look that much at the sources that make it work.